api/v1beta1/awscluster_types.go

```diff
@@ -177,6 +177,11 @@ type AWSLoadBalancerSpec struct {
 	// +optional
 	Subnets []string `json:"subnets,omitempty"`
 
+	// ClassicELBProtocol sets the protocol type for classic ELB health check target
```

Comment:
Since it is defaulted to `SSL` when left empty, should add a comment here.

Reply:
Done